compiler/codegen/Relocation.hpp

```diff
@@ -119,6 +119,7 @@ class LabelRelocation : public TR::Relocation
    TR::LabelSymbol *getLabel()                  {return _label;}
    TR::LabelSymbol *setLabel(TR::LabelSymbol *l) {return (_label = l);}
 
+   void assertLabelDefined();
```

Comment:
This is defined as a public API on the `LabelRelocation` class. Do you think just using a static function in the compilation unit (Relocation.cpp) may be a better solution to avoid exposing this implementation detail?

Reply:
I considered making it protected or a static function, but it just didn't seem that important to limit access to it. I can easily change it if you prefer though

Reply:
Yes let's do that. As we move towards defining a compiler API the less undocumented public APIs we expose the better. If we don't envision subclasses or consumers of the class using the API then we should try not to expose it.

Reply:
> If we don't envision subclasses or consumers of the class using the API

All of the uses are in subclasses, so would you prefer the static function or the protected method?

Reply:
I'm ok with either or. `protected` function seems easiest as it's a one line change.

Reply:
Ok, I've amended the commit to make the method protected